examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -568,17 +592,56 @@ angular.module('cloudberry.common', ['cloudberry.mapresultcache'])
           case "sample":
             cloudberryService.commonTweetResult = result.value[0];
             break;
+          case "timeSeriesRequest":
+            var prefix = Object.freeze({
```

Comment:
As far as I know, the stateID, countyID and cityID will not overlap with each other @waans11 can confirm that? it might not be necessary for this prefix? Then might save some space in case data size increases.